gsy_framework/data_classes.py

```diff
@@ -158,9 +160,9 @@ def serializable_dict(self) -> Dict:
         }
 
     @staticmethod
-    def from_json(json_dict) -> "TraderDetails":
```

Comment:
Are you certain that this will not break things here ? I somehow cannot shrug the feeling that this is there for a reason, that's why I would recommend to check for occurrences of `from_json` in gsy-e / gsy-web if you have not done so already.

Reply:
I had a thorough look even before and found out, that this function is only used in this module and thought to rename it accordingly, because I was confused by the name of the input parameter. either it is a json string or a json serialisable dict right?

Reply:
Since you checked it out already then it is fine by me, it can stay as is.